mlir/test/Target/LLVMIR/openmp-private.mlir

```diff
@@ -0,0 +1,21 @@
+// Test code-gen for `omp.parallel` ops with delayed privatizers (i.e. using
+// `omp.private` ops).
+
+// RUN: mlir-translate -mlir-to-llvmir -split-input-file %s | FileCheck %s
+
+llvm.func @parallel_op_1_private(%arg0: !llvm.ptr) {
+  omp.parallel private(@x.privatizer %arg0 -> %arg2 : !llvm.ptr) {
+//    %0 = llvm.load %arg2 : !llvm.ptr -> f32
```

Comment:
Intentional comment?

Reply:
Same, debugging and experimentation left-over. See my comment.